pkg/i18n/english.go

```diff
@@ -128,6 +128,13 @@ type TranslationSet struct {
 	LcNextScreenMode string
 	LcPrevScreenMode string
 	FilterPrompt     string
+
+	FocusProjects   string
+	FocusServices   string
+	FocusContainers string
+	FocusImages     string
+	FocusVolumes    string
+	FocusNetwork    string
```

Comment:
```suggestion
	FocusNetworks    string
```
Should be plural for consistency

Reply:
@jesseduffield changes has been addressed.